src/components/Form/FormFields/FormField.tsx

```diff
@@ -14,7 +16,7 @@ type LabelProps = {
 
 export const FieldLabel = (props: LabelProps) => {
   return (
-    <label
+    <Label
```

Comment:
instead of modifying this, directly use shadcn's components in the forms. components in `FormFields` are not going to be used in newer codebase.

Reply:
i didn't get what is to be done.

are you suggesting to use [https://ui.shadcn.com/docs/components/form](https://ui.shadcn.com/docs/components/form)

Reply:
even input components from shadcn